Sources/Subs.php

```diff
@@ -6613,6 +6613,8 @@ function build_query_board($userid)
 	else
 		$query_part['query_wanna_see_board'] = '(' . $query_part['query_see_board'] . ' AND b.id_board NOT IN (' . implode(',', $ignoreboards) . '))';
 
+	$query_part['query_see_message'] = str_replace('b.', 'm.', $query_part['query_see_board']);
```

Comment:
For the sake of clarity, it would be nice to rename `query_see_message` to `query_see_message_board`. That would make it easier for future developers to understand exactly what this does.